<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlaveMng</class>
 <widget class="QMainWindow" name="SlaveMng">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>设备ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_channle">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>读取</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>从机基本设置</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>从机温度通道设置</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_up_2">
        <property name="text">
         <string>默认值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_up_4">
        <property name="text">
         <string>所有参数下发</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_up">
        <property name="text">
         <string>下设所有从控</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_up_3">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>参数名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>默认值</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>实际值</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>格式</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>下设</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>读取</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
